apps/native-component-list/src/screens/ModulesCore/ModulesBenchmarksScreen.ios.tsx

```diff
@@ -0,0 +1,216 @@
+import { useTheme } from 'ThemeProvider';
```

Comment:
That screen should also work on Android. Why did you make it iOS only?

Reply:
interesting, for me, bundling was failing on android. Let me fix it